### ISSUE TITLE
Way to Bill Players

### ISSUE THREAD

**Issue by FragOutGR** [renzuzu/renzu_customs#29]
Spoke to Renzuzu about this but I think it would be nice to have a way to bill the player directly for the cost of the upgrades rather than the current setup for it, where the mechanic performing the work is the only one who can initially take the bill.

**Comment by renzuzu** [comment]
before starting with the idea.

do i have to create renzu_billings for this 🐌 ( dependency )
or do you prefer a standalone built in for this?

**Comment by FragOutGR** [comment]
I think either way would work, however built in as a config option (for people who wanted to disable it) would be more ideal.
